### ISSUE TITLE
Improve Sigma support

### ISSUE THREAD

**Issue by jaegeral** [google/timesketch#1199]
To track it here, there is a basic support for sigma in TS, but the vast majority of rules from the Sigma project would not run out of the box.

Reasons for that:
- missing field mapping (esp. if sketch was created based on plaso data)

Some todos on that side:
- [x] Add documentation how to use it / how to create rules
- [x] create basic field mappings to cover plaso parsed evtx files
- [x] Add a story / aggregation after it was ran, to be able to get an idea of what is found

**Comment by jaegeral** [comment]
- [ ] Provide links to sample data

**Comment by jaegeral** [comment]
-[ ] have a folder only for internal rules that people could deploy

**Comment by jaegeral** [comment]
- [ ] Ignore deprecated folders in sigma rule folder, otherwise it would include https://github.com/Neo23x0/sigma/tree/master/rules/windows/deprecated which isncludes a rule that causing to many tags

**Comment by jaegeral** [comment]
There is an issue that IN some rules, it is checking for 
```
Message.content = ""
```

When setting 
```
Message : xml_string
```
 in the mapping file, it will end in a rule that searches for:
```
xml_string.content
```
Will of course fail, this will be adressed with a small hack, that is not really nice.

**Comment by jaegeral** [comment]
- [ ] Set a wait timer in case to many Timesketch queries are caused for ES.

**Comment by jaegeral** [comment]
- [ ] Fix issue with rule: 2b30fa36-3a18-402f-a22d-bf4ce2189f35 to generate to many hits.
